Packs/PhishingURL/Scripts/DBotPredictURLPhishing/DBotPredictURLPhishing.yml

```diff
@@ -59,6 +59,19 @@ args:
   - 'True'
   required: false
   secret: false
+- auto: PREDEFINED
+  description: Reliability of the source providing the intelligence data.
+  defaultValue: A+ - 3rd party enrichment
```

Comment:
Let's validate with @bakatzir that this indeed should be the default

Reply:
indeed, A+.